doc/lua_api.md

```diff
@@ -1470,7 +1470,7 @@ Look for examples in `games/devtest` or `games/minetest_game`.
       'Connected Glass'.
 * `allfaces`
     * Often used for partially-transparent nodes.
-    * External and internal sides of textures are visible.
+    * External sides of textures are visible.
```

Comment:
Was the documentation wrong before or why this change?

Reply:
It never showed the internal textures of the block from the inside, only those of other blocks.

Reply:
"Internal" is meant as in "internal" faces between allfaces nodes aren't culled, not backfaces generally. See Krock's screenshot: 
![Screenshot from 2024-09-20 15-02-53](https://github.com/user-attachments/assets/1fda4958-ab68-450a-b7cd-6d435a37c901)

I agree that the current wording is poor, but just removing it isn't right and would leave the user wondering what this drawtype does at all. After all external sides are visible for regular nodes just as well.

Reply:
@appgurueu I updated the wording